### PR TITLE
krakenfutures - add historical fetchTrades

### PR DESCRIPTION
**Pull request by ttodua**
till date, this used:
https://docs.futures.kraken.com/#http-api-trading-v3-api-instrument-details-get-instrument-status
which only supported last 7 days.

uses endpoint which support historical trades data:
https://docs.futures.kraken.com/#http-api-history-market-history-get-public-execution-events

```
 npm run cli.js krakenfutures fetchTrades BTC/USD:USD 165000000000 2 


> ccxt@4.2.66 cli.js
> node ./examples/js/cli.js krakenfutures fetchTrades BTC/USD:USD 165000000000 2

2024-03-11T11:32:37.258Z
Node.js: v20.10.0                                                           
CCXT v4.2.66
krakenfutures.fetchTrades (BTC/USD:USD, 165000000000, 2)
2024-03-11T11:32:38.425Z iteration 0 passed in 932 ms

                                  id |      symbol |     timestamp |                 datetime | side | takerOrMaker | price | amount |    cost | fees
-----------------------------------------------------------------------------------------------------------------------------------------------------
5c680b49-61d6-49fe-917f-db56509d3053 | BTC/USD:USD | 1648029833450 | 2022-03-23T10:03:53.450Z |  buy |        taker | 42175 |  0.005 | 210.875 |   []
83211033-d6f6-4534-b711-91280c82c8fd | BTC/USD:USD | 1648029833498 | 2022-03-23T10:03:53.498Z |  buy |        taker | 42175 | 0.0015 | 63.2625 |   []
2 objects
2024-03-11T11:32:38.425Z iteration 1 passed in 932 ms
```

### CODE REVIEW

**Comment by carlosmiei** [comment]
@ttodua Can you add a static test?

**Comment by ttodua** [comment]
yes 1 min

**Comment by ttodua** [comment]
done

File: ts/src/krakenfutures.ts [hunk 1]
```diff
@@ -719,47 +723,141 @@ export default class krakenfutures extends Exchange {
...
-        //            ...
-        //        ],
-        //        "serverTime": "2022-03-18T06:39:18.056Z"
-        //    }
-        //
-        const response = await this.publicGetHistory (this.extend (request, params));
-        const history = this.safeValue (response, 'history');
-        return this.parseTrades (history, market, since, limit);
+        let until = undefined;
+        [ until, params ] = this.handleParamInteger (params, 'until');
```
Comment (carlosmiei) [inline ts/src/krakenfutures.ts:727]: we have already `handleUntilAndParams`

Comment (ttodua) [reply ts/src/krakenfutures.ts]: done

**Pull request merged**